include/swift/AST/Attr.h

```diff
@@ -2226,6 +2226,28 @@ class BackDeployAttr: public DeclAttribute {
   }
 };
 
+/// The `@_documentation(...)` attribute, used to note a "category" for a
```

Comment:
Is the term "category" still correct given the latest changes?

Reply:
It's not, good catch 😅